### PR TITLE
Updated and reworded with few fixes

### PR DESCRIPTION
**Pull request by ashish-billore**
**Which issue(s) this PR fixes**:

Fixes #
None

### CODE REVIEW

File: contributors/design-proposals/storage/raw-block-pv.md [hunk 1]
```diff
@@ -560,14 +559,14 @@ spec:
...
-It is important the values that are passed to the container runtimes are valid and support the current implementation of these various runtimes. Listed below are a table of various runtime and the mapping of their values to what is passed from the kubelet.
+It is important the values that are passed to the container runtimes are valid and are supported by the current implementation of these various runtimes. Listed below is a table of various runtimes and the mapping of their values to what is passed from the kubelet.
 
 | runtime engine    | runtime options  | accessMode       |  
 | --------------    |:----------------:| ----------------:|
 | docker/runc/rkt   |  mknod / RWM     | RWO              |
 | docker/runc/rkt   |       R          | ROX              |
 
-The accessModes would be passed as part of the options array and would need validate against the specific runtime engine. 
+The accessModes would be passed as part of the options array and would need to be validated against the specific runtime engine.
```
Comment (vishakhanihore) [inline contributors/design-proposals/storage/raw-block-pv.md]: ~would need to be validated~ would be needed to be validated

File: contributors/design-proposals/storage/raw-block-pv.md [hunk 2]
```diff
@@ -533,8 +532,8 @@ DESCRIPTION:
 * A user wishes to read data from a read-only raw block device, an example might be a database for analytics processing. 
 
 USER:
-* User creates pod and specifies 'readOnly' as a parameter in the persistent volume claim to indicate they would
-like to be bound to a PV with this setting enabled.
+* User creates pod and specifies 'readOnly' as a parameter in the persistent volume claim to indicate he/she would
```
Comment (vishakhanihore) [inline contributors/design-proposals/storage/raw-block-pv.md]: `They` fits better here.

File: contributors/design-proposals/storage/raw-block-pv.md [hunk 3]
```diff
@@ -295,7 +294,7 @@ spec:
 
 DESCRIPTION: 
 
-A developer wishes to enable their application to use a local raw block device as the volume for the container. The admin has already created PVs that the user will bind to by specifying 'block' as the volume type of their PVC.
+A developer wishes to enable his/her application to use a local raw block device as the volume for the container. The admin has already created PVs that the user will bind to by specifying 'block' as the volume type of their PVC.
```
Comment (vishakhanihore) [inline contributors/design-proposals/storage/raw-block-pv.md]: `Their` would be better here.

File: contributors/design-proposals/storage/raw-block-pv.md [hunk 4]
```diff
@@ -33,43 +33,42 @@ This document presents a proposal for managing raw block storage in Kubernetes u
...
+  To ensure backwards compatibility and a phased transition to this feature, the consensus from the community is to intentionally disable
   the volumeMode: Block for both in-tree and external provisioners until a suitable implementation for provisioner versioning has been
   accepted and implemented in the community. In addition, in-tree provisioners should be able to gracefully ignore volumeMode API objects
   for plugins that haven't been updated to accept this value.
   
   It is important to note that when a PV is bound, it is either bound as a raw block device or formatted with a filesystem. Therefore, 
   the PVC drives the request and intended usage of the device by specifying the volumeMode as part of the API. This design lends itself
-  to support of dynamic provisioning by also letting the request initiate from the PVC defining the role for the PV. It also
-  allows flexibility in the implementation and storage plugins to determine their support of this feature. Acceptable values for 
+  to support dynamic provisioning by also letting the request initiated from the PVC defining the role for the PV. It also
```
Comment (vishakhanihore) [inline contributors/design-proposals/storage/raw-block-pv.md]: `request to be initiated`

File: contributors/design-proposals/storage/raw-block-pv.md [hunk 5]
```diff
@@ -33,43 +33,42 @@ This document presents a proposal for managing raw block storage in Kubernetes u
...
   definitions for the block devices will be driven through the PVC and PV definitions. Along with Storage
   Resource definitions, this will provide the admin with a consistent way of managing all storage. 
   The API changes proposed in the following section are minimal with the idea of defining a volumeMode to indicate both the definition
   and consumption of the devices. Since it's possible to create a volume as a block device and then later consume it by provisioning
   a filesystem on top, the design requires explicit intent for how the volume will be used.
   The additional benefit of explicitly defining how the volume is to be consumed will provide a means for indicating the method
   by which the device should be scrubbed when the claim is deleted, as this method will differ from a raw block device compared to a 
-  filesystem. The ownership and responsibility of defining the retention policy shall be up to the plugin method being utilized and is
+  filesystem. The ownership and responsibility of defining the retention policy shall be up to the plugin being utilized and is
```
Comment (vishakhanihore) [inline contributors/design-proposals/storage/raw-block-pv.md]: This is also not required, Plugin method makes sense.

**Comment by mrbobbytables** [comment]
@ashish-billore  I know this is mostly a formatting fix -- but for the most part design proposals are there for historic/archival purposes and not updated. 

They're going to be migrated in the near future to exist alongside their replacement ([Kubernetes Enhancement Proposals](https://github.com/kubernetes/enhancements)). We can leave this open for a bit longer, but we'll be closing out all PRs later this month that touch the design proposals in preparation for their migration.

**Comment by ashish-billore** [comment]
@mrbobbytables thanks for the insight related to this PR. Please do let me know if anything is needed to bring this PR to logical conclusion. Thanks!

**Comment by mrbobbytables** [comment]
@ashish-amarnath thanks for your PR, but at this time we're no longer going to be accepting updates to design-proposals as they're flagged for archival and will soon be moved out of the repo.

With that I'm going to go ahead and close this PR. If you have questions, please feel free to ping me here or in slack.

Thanks!

/close

**Comment by ashish-amarnath** [comment]
@mrbobbytables  I think you intended to tag @ashish-billore  😄

**Pull request closed without merging**